rotkehlchen/accounting/rules.py

```diff
@@ -0,0 +1,170 @@
+from typing import TYPE_CHECKING, Optional
+
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.evm_event import EvmEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.evm.accounting.structures import BaseEventSettings, TxAccountingTreatment
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.db.constants import NO_ACCOUNTING_COUNTERPARTY
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+    from rotkehlchen.chain.evm.accounting.aggregator import EVMAccountingAggregators
+    from rotkehlchen.db.dbhandler import DBHandler
+
+
+def get_accounting_rule_identifier(
```

Comment:
Also I know this will make me hate you for changing all lines again but the name is not what we need. Before it was better. This doesn't need to be an accounting specific, or any rule related identifier. This identifies the event type based on type/subtype/cpt. What we do with this identifier depends on the different modules.

You just happen to care about accounting right now.

```suggestion
def get_event_type_identifier(
```